src/app/event/components.tsx

```diff
@@ -0,0 +1,54 @@
+import type { ReactNode } from "react";
+import { MaterialSymbol } from "react-material-symbols";
+
+export const EventInfoContainer = ({ children }: { children: ReactNode }) => {
+  return (
+    <div className="w-1150px block py-4 px-8 border-2 rounded-3xl mb-10">
+      {children}
+    </div>
+  );
+};
+
+export const StatusFlag = () => {
+  return (
+    <div className="flex w-fit justify-center items-center gap-2 bg-slate-200 py-1 px-3 rounded-lg mb-4">
+      <div className="w-3 h-3 rounded-md bg-slate-400"></div>
+      <span>Published</span>
+    </div>
+  );
+};
+
+export const TicketInfoContainer = ({ children }: { children: ReactNode }) => {
+  return (
+    <div className="w-1150px py-12 px-12 border-2 rounded-3xl">{children}</div>
+  );
+};
```

Comment:
why make this a component? do we think this will repeat?

Reply:
Just to make more clean / readable